error.go

```diff
@@ -660,3 +660,10 @@ func Close(closer io.Closer) Invoker {
 func AppendInvoke(into *error, invoker Invoker) {
 	AppendInto(into, invoker.Invoke())
 }
+
+// AppendFunc is a shorthand for AppendInvoke.
+// It allows using function or method value directly
+// without having to wrap it into an Invoker interface.
```

Comment:
It would be helpful if you could add code samples in the doc so that users can see how to use this, just like the examples above.